packages/wallet/api/test/test-wallet-marshal.js

```diff
@@ -50,7 +50,7 @@ const makeOnChainWallet = board => {
   });
 };
 
-test('makeImportContext preserves identity across AMM and wallet', t => {
```

Comment:
I think we still want a test that `makeImportContext` preserves identity across the AMM and wallet; that was its original motivation.

Reply:
Happy to revert. I thought I was executing your suggestion in https://github.com/Agoric/agoric-sdk/pull/5883#issuecomment-1210060306

Or are you suggesting to keep this change and add something like `test.todo('makeImportContext preserves identity across AMM and wallet')` ?

Reply:
oops... that "better phrasing" suggestion wasn't all that clear; it was about the `t.throws()` message, not the `test()` description.